packages/react-native/ReactCommon/react/renderer/components/view/conversions.h

```diff
@@ -556,6 +556,40 @@ inline void fromRawValue(
   result = transformMatrix;
 }
 
+ inline void fromRawValue(
+     const PropsParserContext &context,
+     const RawValue &value,
+     TransformOrigin &result) {
+  react_native_expect(value.hasType<std::vector<RawValue>>());
+  auto origins = (std::vector<RawValue>)value;
+  react_native_expect(origins.size() == 3);
```

Comment:
Not a huge fan of asserting on things like this. This mean the app will crash if you pass an array of length 2.

Reply:
yeah, same, I saw it at some places in the code and thought would be cool to add it here lol. I think TS does the job or a warning message in JS could also do.